MANIFEST.in

```diff
@@ -1,5 +1,5 @@
+include *.txt
 include VERSION
 include LICENSE
 include README.md
 recursive-include cc_plugin_ugrid *.py
-recursive-include requirements *.txt
```

Comment:
No need for to sets of requirement files. `conda` and `pip` should be compatible, if not we should fix the conda package.